src/main/java/io/openshift/launchpad/ui/osio/BoosterDTO.java

```diff
@@ -0,0 +1,124 @@
+/*
+ * Copyright 2017 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Eclipse Public License version 1.0, available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package io.openshift.launchpad.ui.osio;
+
+import io.openshift.booster.catalog.Booster;
+import io.openshift.booster.catalog.Mission;
+import io.openshift.booster.catalog.Runtime;
+
+/**
+ */
+public class BoosterDTO
```

Comment:
But does it use the Mission and Runtime info? How about replacing the single dropdown list with two dropdown lists (Mission and Runtime) ?